cmd/root.go

```diff
@@ -105,6 +106,7 @@ func init() {
 	rootCmd.Flags().StringP(userArg, "", "", "SAST username")
 	rootCmd.Flags().StringP(passArg, "", "", "SAST password")
 	rootCmd.Flags().StringP(urlArg, "", "", "SAST url")
+	rootCmd.Flags().StringP(queryMapping, "", "", "Path to file query mapping IDs from AST for triage")
```

Comment:
This flag is required if we are exporting triaged results.
A few lines below in this file, there's an instruction to make flags required: `rootCmd.MarkFlagRequired(userArg)`
Here's an example on how to make a flag conditionally required: https://github.com/spf13/cobra/issues/1595

Reply:
Ok